packages/website/ts/utils/utils.ts

```diff
@@ -309,6 +309,17 @@ export const utils = {
             containerId,
         });
     },
+    web3ProviderToString(provider: Web3Provider): string {
+        let parsedProviderName = provider.constructor.name;
+        if (provider.constructor.name === 'MetamaskInpageProvider') {
+            parsedProviderName = 'METAMASK';
+        } else if (provider.constructor.name === 'EthereumProvider') {
+            parsedProviderName = 'MIST';
+        } else if (provider.constructor.name === 'o') {
```

Comment:
Is this because its minified? Will this change on different versions? What does the Cipher Browser constructor look like?

Reply:
Yeah, I just re-tested with a fresh parity install and it's different :/ -- turns out there's just a isParity var. Going to use that instead

Reply:
Not sure about Cipher. Is there any easy way to test it other than spinning up a new web page for it to hit?